diffs/platformref.diff

```diff
@@ -0,0 +1,490 @@
+html/en/beats/libbeat/master/beats-reference.html
+html/en/beats/libbeat/master/breaking-changes-7.0.html
+--- /dev/fd/63
++++ html/en/beats/libbeat/master/breaking-changes-7.0.html
+@@ -278,7 +278,7 @@
+            </a>
+           </h4>
+           <div class="table">
+-           <a id="id-1.8.8.16">
++           <a id="id-1.8.8.13">
+            </a>
+            <p class="title">
+             <strong>
+@@ -931,7 +931,7 @@
+            </div>
+           </div>
+           <div class="table">
+-           <a id="id-1.8.8.17">
++           <a id="id-1.8.8.14">
+            </a>
+            <p class="title">
+             <strong>
+@@ -5664,7 +5664,7 @@
+            </div>
+           </div>
+           <div class="table">
+-           <a id="id-1.8.8.18">
++           <a id="id-1.8.8.15">
+            </a>
+            <p class="title">
+             <strong>
+@@ -6029,7 +6029,7 @@
+            </div>
+           </div>
+           <div class="table">
+-           <a id="id-1.8.8.19">
++           <a id="id-1.8.8.16">
+            </a>
+            <p class="title">
+             <strong>
+@@ -6346,7 +6346,7 @@
+            </div>
+           </div>
+           <div class="table">
+-           <a id="id-1.8.8.20">
++           <a id="id-1.8.8.17">
+            </a>
+            <p class="title">
+             <strong>
+@@ -7367,7 +7367,7 @@
+            </div>
+           </div>
+           <div class="table">
+-           <a id="id-1.8.8.21">
++           <a id="id-1.8.8.18">
+            </a>
+            <p class="title">
+             <strong>
+@@ -8068,7 +8068,7 @@
+            </div>
+           </div>
+           <div class="table">
+-           <a id="id-1.8.8.22">
++           <a id="id-1.8.8.19">
+            </a>
+            <p class="title">
+             <strong>
+@@ -8700,7 +8700,7 @@
+            The Auditbeat JSON data types produced by the output have been changed to align with the data types used in the Elasticsearch index template.
+           </p>
+           <div class="table">
+-           <a id="id-1.8.8.26">
++           <a id="id-1.8.8.22">
+            </a>
+            <p class="title">
+             <strong>
+html/en/beats/libbeat/master/breaking-changes-7.1.html
+html/en/beats/libbeat/master/breaking-changes-7.2.html
+html/en/beats/libbeat/master/breaking-changes-7.3.html
+html/en/beats/libbeat/master/breaking-changes.html
+html/en/beats/libbeat/master/community-beats.html
+html/en/beats/libbeat/master/config-file-format-cli.html
+html/en/beats/libbeat/master/config-file-format-env-vars.html
+html/en/beats/libbeat/master/config-file-format-namespacing.html
+--- /dev/fd/63

```

Comment:
all backslash issues in this diff are fixed in https://github.com/elastic/beats/commit/ecc1d726980ed8651522bf3a3c8bee0aa4b0ee5c